js/scenes/Home.js

```diff
@@ -104,10 +105,16 @@ export class Home extends Component {
           source={{ uri: imageUri }}
           resizeMode="cover"
           style={styles.image}>
-          <Star
-            value={false}
-            onValueChange={() => {}}
-            style={styles.star}/>
+          <View style={styles.topBar}>
+            <Profile
```

Comment:
Is it worth having a separate component for this? IMO it should be a detail of a `ListItem` component, unless we see somewhere else it will be used?